src/crafting.cpp

```diff
@@ -72,7 +72,7 @@ void load_recipe( JsonObject &jsobj, const std::string & /* src */, bool uncraft
     std::string result = jsobj.get_string( "result" );
     std::string category = uncraft ? "CC_NONCRAFT" : jsobj.get_string( "category" );
     int time = jsobj.get_int( "time" );
-    int difficulty = jsobj.get_int( "difficulty" );
+    int difficulty = uncraft ? 0 : jsobj.get_int( "difficulty" );
```

Comment:
Uncraft recipes can be difficult. This makes sense when the object is hard to dismantle properly - for example a robot.